### DISCUSSION TITLE
Custom Client  List view enhancement

### DISCUSSION THREAD

**Discussion by HollaTheWoodFairy** [rustdesk/rustdesk-server-pro#476] [Feature Request]
I think it would be useful to have two new columns in the overview list of created custom clients in the RustDesk Console.
- Date (shows the date when the specific client was created)
- Version (shows the version of the created client - e.g. 1.2.6 - so it's easier to see which one hast to be recreated)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/discussions/441
